Userland/Libraries/LibPDF/Filter.cpp

```diff
@@ -315,7 +314,11 @@ PDFErrorOr<ByteBuffer> Filter::decode_ccitt(ReadonlyBytes, RefPtr<DictObject> de
     (void)black_is_1;
     (void)damaged_rows_before_error;
 
-    return Error::rendering_unsupported_error("CCITTFaxDecode Filter is unsupported");
+    if (k < 0)
+        return Error::rendering_unsupported_error("CCITTFaxDecode Filter Group 4 is unsupported");
+    if (k == 0)
+        return Error::rendering_unsupported_error("CCITTFaxDecode Filter Group 3, 1-D is unsupported");
```

Comment:
We do have support for that :)
But I guess we will add that when we find a PDF that uses it.